README.md

```diff
@@ -1849,24 +1853,27 @@ Which come from Cordova?
 Because startups are so much smaller than many established companies, and because they may
 grow quickly, a few other things are worth remembering when negotiating with startups:
 
-- **Cash versus equity**: If your risk tolerance is reasonably high, it’s reasonable to ask for
-  an offer with more equity and less cash.
+- **Cash versus equity**: If your
+  [risk tolerance](https://www.investopedia.com/terms/r/risktolerance.asp) is reasonably high,
+  it’s reasonable to ask for an offer with more equity and less cash.
 
   - 🚧 More discussion (perhaps in the intro to the Guide) about risk.
     What is risk? What is risk tolerance and how do I know if I have it?
     Should I be risk tolerant?
     What can I look for in the company to determine whether it’s worth a “risk”?
 
-  If a company begins to do well, it’ll likely “level up” lower salaries (bringing them
-  closer to market average) even if you got more equity up front.
-  On the other hand, if you ask for more cash and less equity, it’s unlikely you’ll be able
-  to negotiate to get more equity later on, since equity is increasingly scarce over time
-  (at least in a successful company!).
+  If a company begins to do well, it’ll likely
+  [“level up”](https://bothsidesofthetable.com/this-is-how-startups-level-up-after-raising-money-328d17076515)
```

Comment:
We might mention that this link is to a Mark Suster essay on the topic. When we link to more opiniony/experiential/advice pieces, it's good to point out why it's a trustworthy source. His is a recognizable name so would be effective to share! Build equity 😂